### DISCUSSION TITLE
Issues with video autoplay

### DISCUSSION THREAD

**Discussion by joshpetit** [solidjs/solid#1427] [Q&A]
Hey! I recently built a site in solidjs, but I'm not bale to get the videos to auto play. I have the `muted` and `autplay` attributes set, but on mobile you still need to click on a play button to get the video to start playing. 

This is the code to the video: [src](https://github.com/joshpetit/josh.petit.dev/blob/51e50c11a8068fc944bd4b57bb751f5f50a58ec3/src/App.tsx#L43)
And this is the site you can see does not autoplay: [josh.petit.dev](https://josh.petit.dev)

Opening up my site, the video doesn't autoplay, but opening up [this](https://www.w3schools.com/howto/howto_css_fullscreen_video.asp) site the video does auto play.

Some help would be appreciate thank you!

**Comment by atk** [comment]
That's actually a browser feature in all mobile browsers. Even dispatching your own click events (which are then untrusted) won't change that.

There might be an exception if you have a video without audio tracks (muted does not suffice) and also use the `playsinline` attribute if the user has a wifi connection. Not sure if that still works, though.

**Reply by joshpetit** [reply]
Interesting. I removed the video's audio track, muted it, as well as added the "playsinline" attribute, but it still doesn't seem to play? The video files are webm. I would expect there is a way to do this considering it works for [this](https://www.w3schools.com/howto/howto_css_fullscreen_video.asp) site.

Not to sure what else to try

**Reply by atk** [reply]
MP4 maybe?
